src/roman_datamodels/testing/factories.py

```diff
@@ -409,6 +425,102 @@ def create_flat_ref(**kwargs):
     return stnode.FlatRef(raw)
 
 
+def create_dark_ref(**kwargs):
+    """
+    Create a dummy DarkRef instance with valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    **kwargs
+        Additional or overridden attributes.
+
+    Returns
+    -------
+    roman_datamodels.stnode.DarkRef
+    """
+    raw = {
+        "data": _random_array_float32((4096, 4096, 1)),
+        "dq": _random_array_uint16(),
```

Comment:
Why isn't this 4k x 4k?

Reply:
It is, due to the `_random_array_uint16` default for the size argument.